### DISCUSSION TITLE
How to change volume for all tracks

### DISCUSSION THREAD

**Discussion by Danrejk** [Androz2091/discord-player#1518] [Q&A] [answered]
Using `queue.setVolume(50)` only changes the volume for the song that is currently playing and not all of the songs in the queue.
I bet there is an easy fix for that, which I have simply overlooked, but I just can't figure it out.

**Comment by twlite** [comment, marked as answer]
I think this response is late but setting volume like that will set the same volume for all tracks. Though the code you gave belongs to v5. You will have to use `queue.node.setVolume(50)` or you can also set initial volume by passing `volume: amount` to `nodeOptions`
